.github/workflows/sample-application.yml

```diff
@@ -118,6 +118,11 @@ jobs:
           if [[ ${{ matrix.rn-architecture }} == 'new' ]]; then
             perl -i -pe's/newArchEnabled=false/newArchEnabled=true/g' gradle.properties
             echo 'New Architecture enabled'
+          elif [[ ${{ matrix.rn-architecture }} == 'legacy' ]]; then
```

Comment:
I think this is needed since [by default newArchEnabled=true](https://github.com/getsentry/sentry-react-native/blob/main/samples/react-native/android/gradle.properties#L38)

Reply:
Yes, yes.